nss-day-cohort-43/nuttyshell-immortal-bat-syndicate
# Ag user creates articles

# Description
Implemented news/article functionality to load a newsContainer on the DOM and then push/update articles into it by pulling info from the database. User create and delete functionality enabled.

## Type of change

# Testing Instructions
Confirm that both the events and news modules load properly and that you can create and delete articles and that the module responds accordingly.

# testChecklist:
- [x] Both Event and News modules load
- [x] Articles from database are rendered to DOM
- [x] New articles can be created
- [x] Deleted articles are removed from database and DOM

# myChecklist:
